data/common.yaml

```diff
@@ -17,4 +17,5 @@ superset::log_level: DEBUG
 superset::concurrency: 4
 superset::manage_database: true
 superset::ldap_enabled: false
-superset::row_limit: None
\ No newline at end of file
+superset::row_limit: None
+superset::version: present
```

Comment:
prefer:

`superset::version: latest`

Reply:
oh, right! I only looked at puppet generic default values, and started wanting to implement a ternary operator… but (of course!) reading [the manual](https://forge.puppet.com/modules/puppet/python/2.1.1#pythonpip) makes it much easier!